src/engraving/io/xmlreader.cpp

```diff
@@ -390,8 +390,10 @@ void XmlReader::htmlToString(int level, QString* s)
             --level;
             return;
         case QXmlStreamReader::Characters:
-            if (!isWhitespace()) {
```

Comment:
this is a bug in MU3, it causes space to go missing from text in some circumstances, but I don't know when this isWhitespace() ever *is* needed, probably need to check git history to determine why it was added